index.css

```diff
@@ -82,3 +81,149 @@ body {
 .social-media-spacer {
   margin-left: 20px;
 }
+
+.grid-item {
+  display: block;
+  box-shadow: 2px;
+  padding: 5px;
+  width: 300px;
+  border: 2px solid#DFE1E6;
+  border-radius: 10px;
+}
+
+.image-wrapper {
+  display: flex;
+  justify-content: center;
+}
+
+.image-wrapper > img {
+  width: 250px;
+  border-radius: 30px;
+  align-content: center;
+}
+
+.grid-container {
+  margin-top: 150px;
+  display: grid;
+  grid-template-columns: 1fr;
+  margin-left: 15px;
+  row-gap: 70px;
+}
+
+.grid-item > h1 {
+  font-weight: bold;
+  font-size: x-large;
+  color: #172b4d;
+}
+
+.work-details {
+  display: flex;
+  color: #7a869a;
+  align-items: center;
+  font-size: medium;
+  font-weight: bolder;
+}
+
+.first-detail {
+  color: #344563;
+  font-weight: bold;
+}
+
+.spacer {
+  width: 10px;
+  height: 10px;
+  border-radius: 100%;
+  background: #7a869a;
+}
+
+.margin-spacer {
+  margin-left: 10px;
+}
+
+.grid-item > h2 {
+  color: #344563;
+  font-size: 16px;
+  font-weight: 600;
+  padding: 5px;
+}
+
+.grid-item > ul {
+  display: flex;
+  list-style: none;
+  justify-self: flex-start;
+  padding-inline-start: 10px;
+}
+
+.grid-item > ul > li {
+  background-color: #ebebff;
+  padding: 5px 5px 4px 4px;
+  font-size: 20px;
+  color: #6070ff;
+  border-radius: 5px;
+}
+
+button {
+  border: 2px solid#6070ff;
+  font-size: 20px;
+  color: #6070ff;
+  padding: 8px;
+  border-radius: 15px;
+  background-color: white;
+}
+
+.about-myself {
+  padding-top: 60px;
+  margin-left: 20px;
+  margin-right: 20px;
+}
+
+.about-myself > h3 {
+  color: #7f8cff;
+  padding-left: 20px;
+}
+
+.about-myself > p {
+  color: #344563;
+  font-size: 16px;
+  font-weight: 600;
```

Comment:
Please use the right font size for the `font stylings` for the `About Myself`  heading and the right styling for the paragraph too 👍 

figma design 

![image](https://user-images.githubusercontent.com/20537712/235316724-75419057-db1d-4630-b421-44991a4398b2.png)

Your design :

![image](https://user-images.githubusercontent.com/20537712/235316744-2abb98d3-0edb-4807-8124-22e2ae0bd8eb.png)